systemvm/debian/opt/cloud/bin/update_config.py

```diff
@@ -16,18 +16,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import logging
+logging.basicConfig(filename='/var/log/cloud.log', level=logging.INFO, format='%(asctime)s  %(filename)s %(funcName)s:%(lineno)d %(message)s')
+
```

Comment:
not really helpfull, maybe. we can solve the pylint issue by tickering the order, something like:
```suggestion
import sys
import logging
import subprocess
import os
import os.path
import configure
import json

logging.basicConfig(filename='/var/log/cloud.log', level=logging.INFO, format='%(asctime)s  %(filename)s %(funcName)s:%(lineno)d %(message)s')

from subprocess import PIPE, STDOUT
```
Maybe with more `from x import y`'s. Or have some option on pylint.